mods/ra/rules/vehicles.yaml

```diff
@@ -714,6 +714,12 @@ QTNK:
 	SelectionDecorations:
 		VisualBounds: 44,38,0,-4
 	MadTank:
+	Detonates:
+		ChargeDelay: 96
+		DetonationDelay: 42
+		ChargeSound: MADCHRG2.AUD
+		DetonationSound: MADEXPLO.AUD
```

Comment:
That's unrelated to this pr

On 2015-07-28, at 7:58 AM, Zimmermann Gyula <notifications@github.com> wrote:

> In mods/ra/rules/vehicles.yaml:
> 
> > @@ -714,6 +714,12 @@ QTNK:
> >  	SelectionDecorations:
> >  		VisualBounds: 44,38,0,-4
> >  	MadTank:
> > +	Detonates:
> > +		ChargeDelay: 96
> > +		DetonationDelay: 42
> > +		ChargeSound: MADCHRG2.AUD
> > +		DetonationSound: MADEXPLO.AUD
> The sounds should be lowercase imo.
> 
> ―
> Reply to this email directly or view it on GitHub.
>